### PR TITLE
Added Header and footer style 

### PR DESCRIPTION
**Pull request by Monika39-maker**
Please check my videohoro pr first.

### CODE REVIEW

File: client/src/components/VideoHero.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "../App.css";
+// import image1 from "../images/PladeholderImage.jpeg";
+
+function VideoHero({ src, height, width, subtitle, textOverlay }) {
```
Comment (HThanki) [inline client/src/components/VideoHero.js]: Hi Monika, can I check should this be part of the Header and Footer PR?

File: stories/VideoHero.stories.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import VideoHero from "../client/src/components/VideoHero.js";
```
Comment (HThanki) [inline stories/VideoHero.stories.js]: Hi Monlka, same as above, just checking is this part of the header and footer PR?

File: client/src/components/VideoHero.js (hunk 1 above)
Comment (Monika39-maker) [reply client/src/components/VideoHero.js]: you are right. These changes should have been in the videohero branch. I made mistake. can we still merge it, pls?

File: stories/VideoHero.stories.js (hunk 2 above)
Comment (Monika39-maker) [reply stories/VideoHero.stories.js]: you're right. its not. can we still merge it?

File: client/src/components/VideoHero.js (hunk 1 above)
Comment (Monika39-maker) [reply client/src/components/VideoHero.js]: i am goin to remove it from this branch and push it from the videostory branch

File: stories/VideoHero.stories.js (hunk 2 above)
Comment (Monika39-maker) [reply stories/VideoHero.stories.js]: i am goin to remove it from this branch and push it from the videostory branch

**Pull request merged**